decepticonlp/attacks/attack.py

```diff
@@ -0,0 +1,328 @@
+from tqdm import tqdm
+import torch
+
+loss_type_list = [
+    torch.nn.modules.loss.BCELoss,
+    torch.nn.modules.loss.BCEWithLogitsLoss,
+    torch.nn.modules.loss.CrossEntropyLoss,
+    torch.nn.modules.loss.NLLLoss,
+]
+
+
+class CharAttackerWithHuggingFace:
+    """
```

Comment:
Specify here what kind of model it can work with? As in, something like NMT model which takes as input ... and outputs ...

Reply:
Will do